Wordle.Api/Wordle.Api/Wordle.Api/Services/WordService.cs

```diff
@@ -76,5 +77,80 @@ public async Task<string> GetWordOfTheDay(TimeSpan offset, DateTime? date = null
                 }
             }
         }
+
+        public async Task<string[]> GetAllWords() 
+        {
+            var words = await _db.Words.OrderBy(word => word.Text).ToArrayAsync();
+
+            var wordArray = new string[words.Length];
+            var count = 0;
+
+            foreach (var word in words) 
```

Comment:
A loop shouldn't be necessary here as you're already grabbing the List using .ToArrayAsync()

Reply:
Awesome, thank you!